src/app/Services/MessageService.ts

```diff
@@ -97,6 +98,26 @@ export class MessageService {
                         this.timerService.destructTime, 'info');
                 }
                 break;
+            case EventType.NewMemberEvent:
+                evt = ev as NewMemberEvent;
+                if (this.conversation && this.conversation.id === evt.conversationId) {
+                    this.conversationApiService.ConversationDetail(evt.conversationId)
```

Comment:
I guess that you don't have to call API again. For Stargate already gave everything you need.

Reply:
I wanted to do this. However, Stargate just sent `KahlaUser`, but not `UserGroupRelation`

Reply:
Because when a user leaves, the `UserGroupRelation` instance does not exist anymore.

Reply:
Maybe we should make NewMemberEvent return `UserGroupRelation`, in this way, we don't have to call `ConversationDetail` api again.